src/qtiCommonRenderer/renderers/interactions/TextEntryInteraction.js

```diff
@@ -111,7 +111,9 @@ var render = function render(interaction) {
                 messageType = 'info';
             }
 
-            showTooltip($input, messageType, message);
+            if (count) {
```

Comment:
Are you sure we need remove tooltip here? 
First, it is an informational message:
![image](https://user-images.githubusercontent.com/25976342/79564029-9165ea00-80b6-11ea-8d58-d3b23bbc2b21.png)
 second, with your fix we have an issue when type smth and then remove
![image](https://user-images.githubusercontent.com/25976342/79563767-13a1de80-80b6-11ea-9fac-557035a78126.png)

Reply:
According to @habemuscode it was discussed with Kristina, although, I'd say it should be approved by UX at least, as it's a decision, affecting the whole product,

Reply:
I check again and as long as we will not implement the new indulgent and immediate patterns, the correct behaviour is show tooltip in all cases except error for length = 0 

Working on it

Reply:
![imagen](https://user-images.githubusercontent.com/34692207/79580796-d8110f80-80c9-11ea-8316-3966c343fb99.png)

Is the correct behaviour, so I just refactor the code for the part about length = 0 and type = warning

Reply:
Working on tests

Reply:
FIxed